core/scripts/go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/smartcontractkit/chainlink-common v0.9.6-0.20251007140624-a242e4bbd3dc
 	github.com/smartcontractkit/chainlink-data-streams v0.1.5
 	github.com/smartcontractkit/chainlink-deployments-framework v0.54.0
-	github.com/smartcontractkit/chainlink-evm v0.3.4-0.20251007172225-ba2f4b5ef962
```

Comment:
Can cross check the evm branch here: https://github.com/smartcontractkit/chainlink-evm/commit/edee32e8ca8ad3bda6142d014c43688aad97c088
Parent is `ba2f4b5ef962`